operator/deploy/olm-catalog/ibm-spectrum-scale-csi-operator/2.0.1/csiscaleoperators.csi.ibm.com.crd.yaml

```diff
@@ -0,0 +1,206 @@
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  labels:
+    app.kubernetes.io/instance: ibm-spectrum-scale-csi-operator
+    app.kubernetes.io/managed-by: ibm-spectrum-scale-csi-operator
+    app.kubernetes.io/name: ibm-spectrum-scale-csi-operator
+    release: ibm-spectrum-scale-csi-operator
+  name: csiscaleoperators.csi.ibm.com
+spec:
+  additionalPrinterColumns:
+    - JSONPath: .status.conditions[?(@.type="Ready")].status
+      description: The Nginx Status
```

Comment:
is this description correct, because there is nothing related to nginx in this crd yaml file ?